utils.py

```diff
@@ -0,0 +1,57 @@
+from typing import Set
+import foodfuncs as ff
+
+def new_recipe_prompt():
+    """Runs a script in which the user can enter a recipe and check if it is correct.
+    If it is correct, the recipe is inserted into the database."""
+
+    name = input("Please enter the name of your recipe:  ").lower()
+    instruct = input("Please type the instructions for your recipe:  ")
+    difficulty = int(input("Using numbers 1-3, how difficult is this recipe?  "))
+    while difficulty < 1 or difficulty > 3:

```

Comment:
This might be nitpicking but I think that you should allow the user to have an exit statement here, in case they want to go back to the main prompt without force stopping the code